dis_013_volcano_eruptions/README.md

```diff
@@ -1,9 +1,11 @@
 ## Volcano Eruptions Dataset Near Real-time Script
 This file describes the near real-time script that retrieves and processes the [Volcanoes of the World Database - Eruptions dataset](http://volcano.si.edu/search_eruption.cfm) for [display on Resource Watch](https://resourcewatch.org/data/explore/dis013-Volcanoes-CSV).
 
-{Describe how the original data came from the source.}
+This dataset was provided by the source as a XML file. This XML was first converted into a JSON and then transformed into a table so that it could be uploaded to Carto. In order to transform the data from the JSON to the data table used by Resource Watch, the following changes were made:
+- The '{volcano.si.edu}GeoLocation' variable from the '{volcano.si.edu}Smithsonian_VOTW_Holocene_Eruptions' feature of the JSON was used to create the geometry shown on Resource Watch.
+- A unique ID for each event was created using the '{volcano.si.edu}Eruption_Number' variable of the '{volcano.si.edu}Smithsonian_VOTW_Holocene_Eruptions' feature of the JSON. This was stored in a new column called 'Eruption_Number'.
 
-{Describe the steps used to process the data, e.g., "convert variable X from the original netcdf file to a tif to upload to Google Earth Engine."}
+The data shown on Resource Watch can be found in the 'Volcano_Name' variable of the '{volcano.si.edu}Smithsonian_VOTW_Holocene_Eruptions' feature of the JSON.
```

Comment:
We aren't visualizing the name in any way. We are just showing the point locations of the eruptions.
```suggestion
```